app/src/main/res/values/strings.xml

```diff
@@ -549,6 +549,9 @@ Before uploading your changes, the app checks with a &lt;a href=\"https://www.we
     <string name="pref_summaryOff_show_notes_not_phrased_as_questions">"Ignoring notes not phrased as questions"</string>
 
     <string name="pref_title_quests_restore_hidden">Restore hidden quests</string>
+    <string name="pref_title_quests_restore_hidden_summary_none_hidden">You don’t have any hidden quests.</string>
+    <string name="pref_title_quests_restore_hidden_summary_one_hidden">1 quest is currently hidden.</string>
+    <string name="pref_title_quests_restore_hidden_summary_multiple_hidden">%1$d quests are currently hidden.</string>
```

Comment:
I'd like to reduce the number of necessary strings (=thus, effort for translators)

How about `Number of hidden quests: %d`